blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -62,51 +60,32 @@
 
 public class HttpMetricsIngestionServer {
     private static final Logger log = LoggerFactory.getLogger(HttpMetricsIngestionServer.class);
-    private static TimeValue DEFAULT_TIMEOUT = new TimeValue(5, TimeUnit.SECONDS);
-    private static int WRITE_THREADS = Configuration.getInstance().getIntegerProperty(CoreConfig.METRICS_BATCH_WRITER_THREADS); // metrics will be batched into this many partitions.
-
+    private static TimeValue DEFAULT_TIMEOUT = new TimeValue(1, TimeUnit.MINUTES);
```

Comment:
This might have a side effect of making ingestion slower. Is this change necessary because, now some operations in the processor chain happen synchronously and they need this limit raised to finish?